### ISSUE TITLE
Document for configuring column format

### ISSUE THREAD

**Issue by hoangphucITJP** [man-group/dtale#577]
As mentioned in:
https://github.com/man-group/dtale/issues/519#issuecomment-868474112

I see that we can pass column formating this way:
```
global_state.set_settings(d._data_id, {'columnFormats': formatting})
```

But looking through all the available documents. I can't find any guide on how to use the `columnFormats`.
Please give me some hints. Or could it be that I miss something?

The cause that arises this issue is that I want to programmatically turn on the heatmap for the columns. Can you advise on how to do it

**Comment by aschonfeld** [comment]
@hoangphucITJP I will add documentation on the different settings options and let you know. 

As for turning on the heatmap programmatically that does not exist right now. I did just add the ability to add "Range Highlights" programmatically though. So it shouldn't be hard for me to add the other highlighting options as well:
* Heatmap
* Highlight Dtypes
* Highlight Missing
* Highlight Outliers
I will try to include this in the next release and will let you know when it's ready

**Comment by aschonfeld** [comment]
@hoangphucITJP I have added the ability to set a `background_mode` when calling `dtale.show` in the the version I just released v1.58.1 and I have added the documentation you were looking for [here](https://github.com/man-group/dtale#instance-settings).  Hope this helps. Also, if you haven't already, please toss your ⭐ on the repo.  Thanks 🙏

**Comment by hoangphucITJP** [comment]
> @hoangphucITJP I have added the ability to set a `background_mode` when calling `dtale.show` in the the version I just released v1.58.1 and I have added the documentation you were looking for [here](https://github.com/man-group/dtale#instance-settings). Hope this helps. Also, if you haven't already, please toss your star on the repo. Thanks pray

Cool! Thanks for your effort :+1:

**Comment by hoangphucITJP** [comment]
Hi. I pass `background_mode='heatmap-col'` into `dtale.show` but it does not work.
Maybe you miss passing the param here? https://github.com/man-group/dtale/blob/cbb89ee765357bffd3a37828b56f99f3d150b205/dtale/app.py#L684-L702

**Comment by aschonfeld** [comment]
@hoangphucITJP sorry about that.  Guess thats what I deserve for working too fast.  Just released v1.58.2 which fixes this issue

**Comment by hoangphucITJP** [comment]
* **heatmap-col**: turn on heatmap for all numeric columns where the colors are determined by the range of values in the column

I used this, but the side-effect is it hides the non-numeric columns. Is that intended behavior, @aschonfeld ?
